Makefile

```diff
@@ -22,14 +22,17 @@ E2E_TESTS ?= ./test/e2e/*.bats
 
 E2E_PVC ?= test/e2e/resources/pvc-maven.yaml
 E2E_SECRET ?= test/e2e/resources/secret-maven.yaml
-E2E_PROXY_SECRET ?= test/e2e/resources/secret-proxy-maven.yaml                #
-E2E_CONFIGMAP ?= test/e2e/resources/configmap-proxy-maven.yaml                #
+E2E_PROXY_SECRET ?= test/e2e/resources/secret-proxy-maven.yaml                
+E2E_CONFIGMAP ?= test/e2e/resources/configmap-proxy-maven.yaml                
 E2E_MAVEN_PARAMS_REVISION ?= master
 E2E_MAVEN_PARAMS_URL ?= https://github.com/Aneesh-M-Bhat/shopping-cart-test-java
 E2E_TEST_DIR ?= ./test/e2e
 E2E_MAVEN_PARAMS_SERVER_SECRET ?= secret-maven
-E2E_MAVEN_PARAMS_PROXY_SECRET ?= secret-proxy-maven                               #
-E2E_MAVEN_PARAMS_PROXY_CONFIGMAP ?= config-proxy-maven                              #
+E2E_MAVEN_PARAMS_PROXY_SECRET ?= secret-proxy-maven                               
+E2E_MAVEN_PARAMS_PROXY_CONFIGMAP ?= config-proxy-maven                              
+
+
+E2E_JIB_MAVEN_IMAGE ?= docker.io/aneeshmbhat/tekton-maven:latest
```

Comment:
why this image :thinking:

Reply:
that is where it gets published changing now.

Reply:
its just for test.

Reply:
empty docker repo with latest tag for it to get published to